index.js

```diff
@@ -1,18 +1,233 @@
+  );
+  let secretNumber = await ask(
+    "What's your secret number?\nI won't peek, I promise...\n"
+  );
+  console.log("You entered: " + secretNumber);
+  //First calculation of users given number vs the random number generated from the first CPU guess. 
+  if (randomNum !== secretNumber) {
+    console.log(
+      "Darn it! I thought for sure it was like 99.9(cause I bet you're pretty smart like that..) Time for me to get serious."
+    );
+  }
+  //Setup for the CPU to start making guesses
+  let x = secretNumber;
+  let advancedCpuThingz = function (arr, x, start, end) {
+    // Base Condition
+    if (start > end) return false;
+    // Find the middle index
+    let mid = Math.floor((start + end) / 2);
+    console.log(mid);
+    // Compare middle with given variable
+    if (arr[mid] === x) return true;
+    // If guess at middle is greater than x,
+    // continue search in the left half of the middle
+    if (arr[mid] > x) return advancedCpuThingz(arr, x, start, mid - 1);
+    // If guess at mid is smaller,
+    // continue search in the right half of mid
+    else return advancedCpuThingz(arr, x, mid + 1, end);
+  };
+
+  //Second phase of game. Should first generated random number guess fail, guessing function will be implemented
+  nextStep();
+  async function nextStep() {
+    console.log("I think I got it now you smarty pants!");
+    let comment = await ask(`It wouldn't happen to be ${randomNum} would it? `);
+
+    if (randomNum === secretNumber) {
+      console.log(
+        `You don't even need to respond. I know it was ${secretNumber}! I...am a legend!`
+      );
+      stepfive();
+    } else if (comment == "yes") {
+      console.log(
+        "Hey man. Cheatin ain't cool and neither are cheaters. Try again when you wanna play fair and square"
+      );
+
+      process.exit();
+    } else {
+      console.log(
+        "Perhaps I underestimated you pal. Time for me to pull out all the stops and figure this out once and for all!!"
+      );
+      let count = 0;
+
+      //Just me trying to be cheeky
+      while (count <= 15) {
+        count += 1;
+        if (count === 1) {
+          console.log("Processing!");
+        } else if (count === 13) {
+          console.log("I think your computer's a bit slow...");
+        } else if (count === 15) console.log("DONE");
+        else {
+          console.log("...");
+        }
+      }
+      advancedCpuThingz(arr, secretNumber, 0, arr.length - 1);
+      console.log(
+        "I think my brain just EXPANDED right now dude! I know the answer is in this list! ;)"
+      );
+      stepThree()
+    }
+  }
+  //Confirmation on whether or not secret number was found from guessing function in amount of tries specified.
+  
+  async function stepThree() {
+    let newComment = await ask(`Your number's in this list! Am I right? `);
+    if (
+      (newComment === "yes") ||
+      (newComment === "Yes") ||
+      (newComment === "Yeah") ||
+      (newComment === "yeah")

```

Comment:
You could create a sanitize function so you can anticipate what the user is inputting and manipulate it. (ie take the user input, `toLowerCase()` it, `trim()` off what you dont need and keep what you want to check against [y]ea, [y]es, [y], [y]ep, [n]o, [n]a, [n]ah, [n]ope, etc. _this will be helpful for the next project when you are expecting longer inputs that do much more like movement and room interaction_